### DISCUSSION TITLE
next-test-api-route-handler (NTARH) failed to import api-utils.js

### DISCUSSION THREAD

**Discussion by k2xl** [Xunnamius/next-test-api-route-handler#843] [General]
When running `npm test` I see:

```
 next-test-api-route-handler (NTARH) failed to import api-utils.js

      This is usually caused by:

        1. Using npm@<7 and/or node@<15, which doesn't install peer deps automatically (review install instructions)
        2. NTARH and the version of Next.js you installed are actually incompatible (please submit a bug report)

      Failed import attempts:

```

`node -v` shows `v16.20.0`
`npm -v` shows `9.8.1`

I'm on Mac OSX using jest. I've tried rm -rf the node_modules and reinstalling but still getting this error.

**Comment by k2xl** [comment]
Actually just solved it.. It was strange because it started happening only recently and only with certain tests. Then I realized I had to enableFetchMocks()

`import { enableFetchMocks } from 'jest-fetch-mock';`
Then call `enableFetchMocks()`

Writing here in case it solves and future people

**Comment by Xunnamius** [comment]
@k2xl Glad to hear it's working. Are you mocking node-fetch (NTARH's fetch function) in your setup? I'm curious what was interfering with the import and why your solution worked.
